x-pack/plugins/index_lifecycle_management/common/types/policies.ts

```diff
@@ -141,6 +143,44 @@ export interface ForcemergeAction {
   index_codec?: 'best_compression';
 }
 
+export interface RollupAction {
+  config: {
+    groups: {
+      date_histogram: {
+        field: string;
+        /**
+         * Deprecated interval specification option. Mutually exclusive with "calendar_interval" and "fixed_interval".
+         */
+        interval?: string;
+        /**
+         * Mutually exclusive with "interval" and "fixed_interval".
+         */
+        calendar_interval?: string;
+        /**
+         * Mutually exclusive with "interval" and "calendar_interval".
+         */
+        fixed_interval?: string;
+        delay?: string;
+        time_zone?: string;
+      };
+      terms?: {
+        fields: string[];
+      };
+      histogram?: {
+        interval?: string;
+        fields: string[];
+      };
+    };
+    metrics?: Array<{ field: string; metrics: string[] }>;
+  };
+  /**
+   * Policy to manage the new, rollup index created by this action.
+   *
+   * Default behavior is to use the policy that created the index to manage the index too.
+   */
+  rollup_policy?: string;
```

Comment:
I think we don't need the policy in the rollup action anymore?

Reply:
At the moment it is still part of the API, we are just not including it in the UI at this time. This comment can be updated though because it is inaccurate -  the rollup index is actually not managed by any policy when it is created by this action.